### PR TITLE
Conscious Language Updates

### PR DESCRIPTION
**Pull request by paulrobinson**
Here's a few 'Conscious Language Updates'. It doesn't cover all the problematic language occurrences, but it does cover those that i could have a go at fixing before it was home time.

The changes are based on the guidance in https://source.redhat.com/aboutredhat/weareredhat/inclusion/diversity__inclusion_wiki/conscious_language_project_faq.

In many cases I didn't know the full intent of the code, so a better alternative might be appropriate. Let me know, and I'll fix.

### CODE REVIEW

File: extensions/elasticsearch-rest-client-common/runtime/src/main/java/io/quarkus/elasticsearch/restclient/common/runtime/graal/Substitute_RestClient.java [hunk 1]
```diff
@@ -38,7 +38,7 @@
 final class Substitute_RestClient {
 
     @Alias
-    private ConcurrentMap<HttpHost, DeadHostState> blacklist;
+    private ConcurrentMap<HttpHost, DeadHostState> denylist;
```
Comment (machi1990) [inline extensions/elasticsearch-rest-client-common/runtime/src/main/java/io/quarkus/elasticsearch/restclient/common/runtime/graal/Substitute_RestClient.java:41]: IIRC, the `Alias` has to exactly have the name of the field that is being referenced. In this case it has to match the: https://github.com/elastic/elasticsearch/blob/master/client/rest/src/main/java/org/elasticsearch/client/RestClient.java#L115 field

**Review by gastaldi — CHANGES_REQUESTED** [review]
It looks good, except for the `@Alias` feature that @machi1990 pointed out in https://github.com/quarkusio/quarkus/pull/19547/files#r693152986

File: extensions/undertow/deployment/src/main/java/io/quarkus/undertow/deployment/UndertowBuildStep.java [hunk 2]
```diff
@@ -248,8 +248,8 @@ public void registerUndertowHandlersConf(BuildProducer<ServletExtensionBuildItem
 
     @BuildStep
     List<IgnoredServletContainerInitializerBuildItem> translateDeprecated(
-            List<BlacklistedServletContainerInitializerBuildItem> old) {
```
Comment (gastaldi) [inline extensions/undertow/deployment/src/main/java/io/quarkus/undertow/deployment/UndertowBuildStep.java:251]: I am not sure this is correct, I think it exists for compatibility purposes, maybe the method can be removed completely along with the `BlacklistedServletContainerInitializerBuildItem` class

File: extensions/elasticsearch-rest-client-common/runtime/src/main/java/io/quarkus/elasticsearch/restclient/common/runtime/graal/Substitute_RestClient.java (hunk 1 above)
Comment (cescoffier) [reply extensions/elasticsearch-rest-client-common/runtime/src/main/java/io/quarkus/elasticsearch/restclient/common/runtime/graal/Substitute_RestClient.java:41]: Yes, so we can't apply this change without a change in the Elastic code source.
@paulrobinson can you revert this one?

**Review by cescoffier — DISMISSED** [review]

File: extensions/undertow/deployment/src/main/java/io/quarkus/undertow/deployment/UndertowBuildStep.java (hunk 2 above)
Comment (gsmet) [reply extensions/undertow/deployment/src/main/java/io/quarkus/undertow/deployment/UndertowBuildStep.java:252]: I removed it.

**Comment by gsmet** [comment]
I fixed the issues and force pushed. Let's see what CI has to say.

File: extensions/elasticsearch-rest-client-common/runtime/src/main/java/io/quarkus/elasticsearch/restclient/common/runtime/graal/Substitute_RestClient.java (hunk 1 above)
Comment (gsmet) [reply extensions/elasticsearch-rest-client-common/runtime/src/main/java/io/quarkus/elasticsearch/restclient/common/runtime/graal/Substitute_RestClient.java]: Reverted.

**Comment by paulrobinson** [comment]
Thanks @gsmet, sorry I didn't get to the changes promptly. Damn those People Managers doing some code!!

**Review by gastaldi — APPROVED** [review]

**Pull request merged**